goofoo_abs.xml.fdm_material

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" version="1.3">
+    <metadata>
+        <name>
+            <brand>GOOFOO</brand>
+            <material>ABS</material>
+            <color>Generic</color>
+            <label>ABS+</label>
+        </name>
+        <GUID>f975e1d4-734f-5a0c-b5ac-6551704c2c37</GUID>
+        <version>2</version>
+        <color_code>#ee51f8</color_code>
+    </metadata>
+    <properties>
+        <density>1.10</density>
+        <diameter>1.75</diameter>
+    </properties>
+    <settings>
+        <setting key="print temperature">240</setting>
+        <setting key="heated bed temperature">80</setting>
+        <setting key="standby temperature">240</setting>
+        <setting key="retraction amount">4.5</setting>
+        <setting key="retraction speed">60</setting>
+        <machine>
+            <machine_identifier manufacturer="GOOFOO" product="goofoo" />
+            <setting key="print cooling">0</setting>
+            <setting key="standby temperature">240</setting>
+            <setting key="retraction speed">60</setting>
+            <setting key="retraction amount">4.5</setting>
+            <setting key="heated bed temperature">80</setting>
+            <setting key="print temperature">240</setting>
+            <hotend id="0.25mm thermoplastic extruder">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="retraction speed">60</setting>
+                <setting key="retraction amount">4.5</setting>
+            </hotend>
+            <hotend id="0.4mm thermoplastic extruder">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="retraction speed">60</setting>
+                <setting key="retraction amount">4.5</setting>
+            </hotend>
+            <hotend id="0.8mm thermoplastic extruder">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="retraction speed">60</setting>
+                <setting key="retraction amount">4.5</setting>
+            </hotend>
+        </machine>
+    </settings>
+</fdmmaterial>
```

Comment:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<fdmmaterial xmlns="http://www.ultimaker.com/material" version="1.3">
    <metadata>
        <name>
            <brand>GOOFOO</brand>
            <material>ABS</material>
            <color>Generic</color>
            <label>ABS+</label>
        </name>
        <GUID>f975e1d4-734f-5a0c-b5ac-6551704c2c37</GUID>
        <version>2</version>
        <color_code>#ee51f8</color_code>
    </metadata>
    <properties>
        <density>1.10</density>
        <diameter>1.75</diameter>
    </properties>
    <settings>
        <setting key="print temperature">240</setting>
        <setting key="heated bed temperature">80</setting>
        <setting key="standby temperature">240</setting>
        <setting key="retraction amount">4.5</setting>
        <setting key="retraction speed">60</setting>
        <machine>
            <machine_identifier manufacturer="GooFoo" product="GooFoo Nova" />
            <setting key="print cooling">0</setting>
            <hotend id="0.2mm Nozzle">
                <setting key="hardware compatible">yes</setting>
            </hotend>
            <hotend id="0.4mm Nozzle">
                <setting key="hardware compatible">yes</setting>
            </hotend>
            <hotend id="0.8mm Nozzle">
                <setting key="hardware compatible">yes</setting>
            </hotend>
        </machine>
    </settings>
</fdmmaterial>
```

The above will work fine ONLY with the printer `goofoo_nova.def.json` and ONLY if you change the "name" and "machine_name" of that definition file to "GooFoo Nova". Now if you want the ABS+ material to properly override values for other printers, you need to add them to this file.